src/components/PageBlocks/RecordBase.vue

```diff
@@ -25,7 +25,7 @@
           {{ field.label || field.name }}
           <hint
             :id="field.fieldID"
-            :text="(field.options.hint || {}).view || ''"
+            :text="((field.options.hint || {}).view || '')"
```

Comment:
Reason for this?

Reply:
there is a lint issue with it that messes up the IDE coloring.